test/functional/test_framework/mempool_util.py

```diff
@@ -25,8 +28,8 @@ def fill_mempool(test_framework, node, miniwallet):
     It will not ensure mempools become synced as it
     is based on a single node and assumes -minrelaytxfee
     is 1 sat/vbyte.
-    To avoid unintentional tx dependencies, it is recommended to use separate miniwallets for
-    mempool filling vs transactions in tests.
+    To avoid unintentional tx dependencies, the mempool filling txs are created with a
+    tagged ephemeral miniwallet instance.
```

Comment:
This moves the responsibility of avoiding unintentional tx dependencies from the caller to this function instead. Is this the right way to understand this?